presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -217,6 +252,7 @@ public void createPartitionedTableAs()
         }
     }
 
+    @Test
```

Comment:
This should not be added, since this is not a test method (though it shouldn't be public, that's a separate fix)